### ISSUE TITLE
Add other antiword parameters

### ISSUE THREAD

**Issue by mayeulk** [ropensci/antiword#5]
It would be great to be able to use other parameters, for instance:
```
-w width
              In text mode this is the line width in characters. A value of zero puts an entire paragraph on a line, useful when the text is to used as input for another wordprocessor.
```
For example -w 0 would be helpfull for extracting text in an NLP pipeline.

**Comment by mayeulk** [comment]
NLP pipelines use antiword R package, for example the 
readtext::readtext() function. readtext() has a general parameter "...":
```
...: additional arguments passed through to low-level file reading
          function, such as ‘file’, ‘fread’, etc.  Useful for
          specifying an input encoding option, which is specified in
          the same was as it would be give to ‘iconv’.  See the
          Encoding section of file for details.
```
This could be used to pass -w 0 to antiword. Currently, a workaround to `readtext(file=full_path_ext)` is this:
```
 if(file_ext=="doc"){
    file_text<-system2(command = "antiword", args = paste0(" -w 0 ", full_path_ext),stdout = TRUE )  
    } else {
    file_text<-readtext(file=full_path_ext)
    }
```
The problem with readtext and antiword without the w parameter is that NLP pipelines often fail with additional line breaks added by antiword.
For example, take the sentence "I spoke to Mr. Walking in the street, I saw him." If antiword splits the sentence after Mr. with a new line, most NLP pipelines will believe there are 2 sentences:
1) "I spoke to Mr."
2)  "Walking in the street, I saw him."
In the original sentence, "Mr. Walking" is a named entity recognised by NLP. In the second sentence, "walking" is the verb "to walk" of which "I" is the subject.

**Comment by mayeulk** [comment]
NLP pipelines use antiword R package, for example the 
readtext::readtext() function. readtext() has a general parameter "...":
```
...: additional arguments passed through to low-level file reading
          function, such as ‘file’, ‘fread’, etc.  Useful for
          specifying an input encoding option, which is specified in
          the same was as it would be give to ‘iconv’.  See the
          Encoding section of file for details.
```
This could be used to pass -w 0 to antiword. Currently, a workaround to `readtext(file=full_path_ext)` is this:
```
 if(file_ext=="doc"){
    file_text<-system2(command = "antiword", 
       args = paste0(" -w 0 ", full_path_ext), stdout = TRUE )  
    } else {
    file_text<-readtext(file=full_path_ext)
    }
```
This workaround does not always work as  help(system2) says: 

> Output lines of more than 8095 bytes will be split.

The problem with readtext and antiword without the w parameter is that NLP pipelines often fail with additional line breaks added by antiword.
For example, take the sentence "I spoke to Mr. Walking in the street, I saw him." If antiword splits the sentence after Mr. with a new line, most NLP pipelines will believe there are 2 sentences:
1) "I spoke to Mr."
2)  "Walking in the street, I saw him."
In the original sentence, "Mr. Walking" is a named entity recognised by NLP. In the second sentence, "walking" is the verb "to walk" of which "I" is the subject.
